### PR TITLE
[Wallet] Add funds with Moonpay widget

### PR DESCRIPTION
**Pull request by annakaz**
### Description

Add the Moonpay widget to the Account page with the option to buy funds (disabled with a feature flag right now). 

Note that the "signMoonpay" firebase function signing the requests is not included in this PR, as I am not sure whether it should be added to the monorepo. It is a straightforward function for HMAC signatures deployed here: https://console.cloud.google.com/functions/details/us-central1/signMoonpay?organizationId=54829595577&project=celo-org-mobile&tab=general
This signing is required if we want to prefill a user's address and is linked to the C Labs Moonpay account. 

### Tested

Tested that the widget comes up. Since we do not yet have production keys I cannot test that I can buy anything (can only go through auth flow), but we will ~~chat with Moonpay on Monday to get this started~~  be able to test this integration by the end of the week after Moonpay integrates on their side.

### Related issues

- Partially addresses #2373

### CODE REVIEW

**Comment by annakaz** [comment]
Screenshot_20200210-133609

**Comment by annakaz** [comment]
Note that build is failing on master (unrelated to this PR)

File: packages/mobile/src/account/FiatExchange.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  componentDidMount() {
+    if (this.props.account) {
+      signMoonpayUrl(this.props.account, this.props.localCurrency)
+        .then((signedUrl) => this.setState({ signedUrl }))
+        .catch(() => this.handleError())
+    }
+  }
+
+  handleError = () => {
+    showError(ErrorMessages.FIREBASE_FAILED)
```
Comment (cmcewen) [inline packages/mobile/src/account/FiatExchange.tsx:78]: Is this the right error?

File: packages/mobile/src/account/FiatExchange.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    headerTitle: i18n.t('accountScreen10:addFunds'),
+  })
+
+  state: State = {
+    signedUrl: '',
+  }
+
+  componentDidMount() {
+    if (this.props.account) {
+      signMoonpayUrl(this.props.account, this.props.localCurrency)
```
Comment (cmcewen) [inline packages/mobile/src/account/FiatExchange.tsx]: Could you make this a separate function called in componentDidMount so that you can use async/await?

**Review by cmcewen — APPROVED** [review]

File: packages/mobile/src/account/FiatExchange.tsx (hunk 1 above)
Comment (annakaz) [reply packages/mobile/src/account/FiatExchange.tsx:78]: Yeah this is an error thrown when Firebase can't sign the URL

**Pull request merged**